app/controllers/users.js

```diff
@@ -55,6 +56,27 @@ exports.getAll = (request, response, next) => {
     })
     .catch(error => {
       logger.error(error);
-      response.status(400).send('Unexpected data base error');
+      response.status(400).send(errors.defaultError('Unexpected data base error'));
     });
 };
+
+exports.createAdmin = (request, response) => {
+  if (request.validationErrors.length > 0)
```

Comment:
I think you should make this validation and return the errors directly on the middleware